tools/ansible/tasks/wsu/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- hosts: localhost
+  vars:
+    project_root: "{{ playbook_dir }}/../../../.."
```

Comment:
I'm fine with this for now but things to keep in mind:
How well this plays well with our CI like downloading files onto pods and then transferring them. One option would be have a flag to skip downloading the binaries and let them build locally and transferred.

Reply:
Good point. That should definitely be addressed in a future commit

Reply:
This indeed looks hairy

Reply:
I am fine with future commit.